extensions/data-transfer/portability-data-transfer-google/src/main/java/org/datatransferproject/datatransfer/google/mediaModels/GoogleMediaItem.java

```diff
@@ -107,7 +114,18 @@ public static PhotoModel convertToPhotoModel(
         albumId.orElse(null),
         false  /*inTempStore*/,
         null  /*sha1*/,
-        mediaItem.getUploadedTime());
+        getCreationTime(mediaItem));
+  }
+
+  private static Date getCreationTime(GoogleMediaItem mediaItem) throws ParseException  {
+    // cannot be empty or null based. Verified the backend code.
+    try {
+      return CREATION_TIME_FORMAT.parse(mediaItem.getMediaMetadata().getCreationTime());
+    } catch (ParseException parseException) {
+      throw new ParseException(String.format("Skipping item transfer since code failed to parse the "
+          + "string %s to get creationTime. Let's look into the dataFormatter",
+          mediaItem.getMediaMetadata().getCreationTime()), 0);
```

Comment:
I think the 0 is not necessary, there's only one wildcard in the String, not sure what the 0 is doing.

Reply:
switched to getErrorOffset